### PR TITLE
[zh] clean up reference page of zh, make it up-to-date

### PR DESCRIPTION
**Pull request by howieyuen**
ref: https://github.com/kubernetes/website/issues/22385

clean up zh reference page

- fix 404 
- update by en page
- optimize title translation of labels-annotations-taints.md

### CODE REVIEW

File: content/zh/docs/reference/_index.md [hunk 1]
```diff
@@ -78,28 +69,30 @@ client libraries:
...
-* [kube-controller-manager](/docs/admin/kube-controller-manager/) - 一个守护进程，它嵌入到了 Kubernetes 的附带的核心控制循环。
-* [kube-proxy](/docs/admin/kube-proxy/) - 可以跨一组后端进行简单的 TCP/UDP 流转发或循环 TCP/UDP 转发。
-* [kube-scheduler](/docs/admin/kube-scheduler/) - 一个调度程序，用于管理可用性、性能和容量。
-
+## 组件参考
+
+* [kubelet](/zh/docs/reference/command-line-tools-reference/kubelet/) - 在每个节点上运行的主 *节点代理* 。kubelet 采用一组 PodSpecs 并确保所描述的容器健康地运行。
+* [kube-apiserver](/zh/docs/reference/command-line-tools-reference/kube-apiserver/) - REST API，用于验证和配置 API 对象（如 pod，服务，副本控制器）的数据。
+* [kube-controller-manager](/zh/docs/reference/command-line-tools-reference/kube-controller-manager/) - 一个守护进程，它嵌入到了 Kubernetes 的附带的核心控制循环。
+* [kube-proxy](/zh/docs/reference/command-line-tools-reference/kube-proxy/) - 可以跨一组后端进行简单的 TCP/UDP 流转发或循环 TCP/UDP 转发。
```
Comment (tengqm) [inline content/zh/docs/reference/_index.md:92]:
```suggestion
* [kube-proxy](/zh/docs/reference/command-line-tools-reference/kube-proxy/) - 可进行简单的 TCP/UDP 流转发或针对一组后端执行轮流 TCP/UDP 转发。
```

File: content/zh/docs/reference/_index.md [hunk 2]
```diff
@@ -78,28 +69,30 @@ client libraries:
...
-* [kubelet](/docs/admin/kubelet/) - 在每个节点上运行的主 *节点代理* 。kubelet 采用一组 PodSpecs 并确保所描述的容器健康地运行。
-* [kube-apiserver](/docs/admin/kube-apiserver/) - REST API，用于验证和配置 API 对象（如 pod，服务，副本控制器）的数据。
-* [kube-controller-manager](/docs/admin/kube-controller-manager/) - 一个守护进程，它嵌入到了 Kubernetes 的附带的核心控制循环。
-* [kube-proxy](/docs/admin/kube-proxy/) - 可以跨一组后端进行简单的 TCP/UDP 流转发或循环 TCP/UDP 转发。
-* [kube-scheduler](/docs/admin/kube-scheduler/) - 一个调度程序，用于管理可用性、性能和容量。
-
+## 组件参考
+
+* [kubelet](/zh/docs/reference/command-line-tools-reference/kubelet/) - 在每个节点上运行的主 *节点代理* 。kubelet 采用一组 PodSpecs 并确保所描述的容器健康地运行。
+* [kube-apiserver](/zh/docs/reference/command-line-tools-reference/kube-apiserver/) - REST API，用于验证和配置 API 对象（如 pod，服务，副本控制器）的数据。
```
Comment (tengqm) [inline content/zh/docs/reference/_index.md:90]:
```suggestion
* [kube-apiserver](/zh/docs/reference/command-line-tools-reference/kube-apiserver/) - REST API，用于验证和配置 API 对象（如 Pod、服务或副本控制器等）的数据。
```

File: content/zh/docs/reference/kubernetes-api/labels-annotations-taints.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 知名标签（Label）、注解（Annotation）和 Taints
+title: 知名标签（Label）、注解（Annotation）和 污点（Taint）
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/labels-annotations-taints.md:2]: 👍

**Review by tengqm — COMMENTED** [review]
see inline

File: content/zh/docs/reference/_index.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/reference/_index.md]: fixed

File: content/zh/docs/reference/_index.md (hunk 2 above)
Comment (howieyuen) [reply content/zh/docs/reference/_index.md]: fully checked next time 😆

**Review by tanjunchen — APPROVED** [review]
/lgtm

**Pull request merged**